Documentation/ReleaseNotes.md

```diff
@@ -144,6 +148,12 @@ Specify the desired raycast provider in the Input System's configuration profile
 
 ![Selecting the Raycast provider](Images/ReleaseNotes/SelectRaycastProvider.png)
 
+**Improved Solvers**
+
+Solvers can now be controlled by the first available hand/controller instead of being locked to left or right handedness. Old value types of *Tracked Target Type* need to be updated to the new non-obsolete values.
```

Comment:
Add a link to the solver breaking changes part to explain how to migrate

Reply:
added link.